### ISSUE TITLE
Support for keyword arguments + Style/HashSyntax: hash_rockets

### ISSUE THREAD

**Issue by thomsbg** [rubocop/rubocop#1451]
My project has the following rule in `.rubocop.yml`:

```
Style/HashSyntax:
  EnforcedStyle: hash_rockets
```

The problem is, I would also like to allow Ruby 2 style keyword arguments. This keeps the syntax for keyword arguments distinct from that of passing a Hash without the `{}`. However, rubocop marks this as a style violation. Is there any way for rubocop to handle this situation?

**Comment by sandstrom** [comment]
I have this issue too. 

Would it be possible to make the cop configurable, such that it wouldn't do this check on arguments?

```sh
test/unit/lib/service_finder_test.rb:62:29: C: Style/HashSyntax: Use hash rockets syntax.
    record2 = create_record(target: '127.0.0.20', port: 80, ttl: 5)
                            ^^^^^^^
```
